test/test.js

```diff
@@ -458,6 +460,27 @@ function checkRefTestResults(browser, id, results) {
   });
 }
 
+function browserTestReportHandler(req, res) {
+  var parsedUrl = url.parse(req.url, true);
```

Comment:
This variable is not used, so it can be removed.

Reply:
Actually, its used in the very next line, so we'll have to keep it I guess